frontend/app/src/locales/en.json

```diff
@@ -614,6 +614,46 @@
     "default_timeframe": "Dashboard graph default timeframe",
     "default_timeframe_description": "Set the default time frame for the dashboard graph. This timeframe will be pre-selected upon login. By default it will remember the previous session's selection."
   },
+  "frontend_settings": {
+    "hint": {
+      "refresh": "Select how often your balances will be automatically refreshed"
+    },
+    "label": {
+      "query_period": "Query period in seconds",
```

Comment:
Perhaps not seconds but minutes as discussed?

Reply:
Query Period = Periodic Query Period